### PR TITLE
Address some minor issues in the release guide

### PR DESCRIPTION
**Pull request by ephraimbuddy**

### CODE REVIEW

File: dev/README_RELEASE_AIRFLOW.md [hunk 1]
```diff
@@ -852,7 +857,7 @@ Documentation for providers can be found in the ``/docs/apache-airflow`` directo
 
     ```shell script
     cd "${AIRFLOW_REPO_ROOT}"
-    breeze build-docs apache-airflow docker-stack --clean-build
+    breeze build-docs --package-filter apache-airflow --package-filter docker-stack --clean-build
```
Comment (higrys) [inline dev/README_RELEASE_AIRFLOW.md:860]:
```suggestion
    breeze build-docs --package-filter apache-airflow --package-filter docker-stack --clean-build
```

No need to add `--package-filter` the original version should work (and even be auto-completable). The `--package-filter` is something of a past now that most people should not use (including release managers).

Comment (potiuk) [inline dev/README_RELEASE_AIRFLOW.md]:
```suggestion
    breeze build-docs --package-filter apache-airflow --package-filter docker-stack --clean-build
```

No need to add `--package-filter` the original version should work (and even be auto-completable). The `--package-filter` is something of a past now that most people should not use (including release managers).

**Review by potiuk — COMMENTED** [review]
Approved with small nit.

**Review by potiuk — APPROVED** [review]

File: dev/README_RELEASE_AIRFLOW.md (hunk 1 above)
Comment (ephraimbuddy) [inline dev/README_RELEASE_AIRFLOW.md]:
```suggestion
    breeze build-docs apache-airflow docker-stack --clean-build
```

**Review by dirrao — APPROVED** [review]

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**